recipes/numcpp/all/conanfile.py

```diff
@@ -11,6 +11,14 @@ class NumCppConan(ConanFile):
     topics = ("python", "numpy", "numeric")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/dpilger26/NumCpp"
+    options = {
+        "use_boost" : [True, False],
+        "use_thread" : [True, False],
```

Comment:
Please rename `use_boost` to `with_boost` as it will add an extra requirement on boost.
Rename `use_threads` to something like `threads`.

Reply:
I will rename these options "use_boost" and  "threads".